lightly/active_learning/agents/agent.py

```diff
@@ -34,78 +38,142 @@ class ActiveLearningAgent:
         >>>
         >>> # make an initial active learning query
         >>> sampler_config = SamplerConfig(n_samples=100, name='initial-set')
-        >>> initial_set = agent.query(sampler_config)
-        >>> unlabeled_set = agent.unlabeled_set
+        >>> initial_set, _ = agent.query(sampler_config)
         >>>
         >>> # train and evaluate a model on the initial set
-        >>> # make predictions on the unlabeled set (keep ordering of filenames)
+        >>> # make predictions on the query set:
+        >>> query_set = agent.query_set
+        >>> # important:
+        >>> # be sure to keep the order of the query set when you make predictions
         >>>
         >>> # create active learning scorer
         >>> scorer = ScorerClassification(predictions)
         >>>
         >>> # make a second active learning query
         >>> sampler_config = SamplerConfig(n_samples=200, name='second-set')
-        >>> second_set = agent.query(sampler_config, scorer)
+        >>> second_set, added_set = agent.query(sampler_config, scorer)
 
     """
 
-    def __init__(self, api_workflow_client: ApiWorkflowClient, query_tag_name: str = None,
+    def __init__(self,
+                 api_workflow_client: ApiWorkflowClient,
+                 query_tag_name: str = None,
```

Comment:
We could make it even easier and cleaner with `query_tag_name: str = "initial-tag"`. Then no checks are necessary if the `query_tag_name is None` and it is documented in code to which tag the query tag defaults.

Reply:
Yes, could do that.

Reply:
@philippmwirth What do you think about that?